packages/warriorjs-cli/src/loadTowers.js

```diff
@@ -0,0 +1,27 @@
+import path from 'path';
+
+import findUp from 'find-up';
+import globby from 'globby';
+import resolve from 'resolve';
+
+import Tower from './Tower';
+
+async function loadTowers() {
+  const towerSearchDir = await findUp('node_modules');
+  const officialTowerPackageJsonPattern = '@warriorjs/tower-*/package.json';
+  const communityTowerPackageJsonPattern = 'warriorjs-tower-*/package.json';
+  const towerPackageJsonPaths = await globby(
+    [officialTowerPackageJsonPattern, communityTowerPackageJsonPattern],
+    { cwd: towerSearchDir },
+  );
+  const towerPaths = towerPackageJsonPaths.map(path.dirname);
+  const towerRequirePaths = towerPaths.map(towerPath =>
+    resolve.sync(towerPath, { basedir: towerSearchDir }),
+  );
+  const towers = towerRequirePaths.map(
+    towerRequirePath => require(towerRequirePath), // eslint-disable-line global-require, import/no-dynamic-require
```

Comment:
Is the `require()` call safe? I saw this same thing being done like this in [other places](https://github.com/kachkaev/prettier/blob/bfbc580af2f15851aff75b59491e03c259ccc718/src/common/load-plugins.js#L77):

```js
eval('require')(towerRequirePath)
```

Reply:
`eval('require')` is used in Prettier because that module is built with Rollup and `eval` prevents stops `towerRequirePath` from an attempt to be glued together with the current module. If you don't have Rollup in your workflow, feel free to just `require(towerRequirePath)`.

Hope this helps! 👋

Reply:
Ah! Makes perfect sense, thanks!